webapp/src/Service/DOMJudgeService.php

```diff
@@ -1228,13 +1228,15 @@ public function getAssetFiles(string $path): array
 
         $results = [];
         foreach (scandir($customDir) as $file) {
-            foreach (static::MIMETYPE_TO_EXTENSION as $extension) {
+            foreach (array_merge(['css'],static::MIMETYPE_TO_EXTENSION) as $extension) {
```

Comment:
We also need .js in that case

Reply:
Also missing space after ,

Reply:
But where did this stop working.... and why?

Reply:
I think when I rewrote everything to not use the symfony DI container detection